### ISSUE TITLE
Subscriptions Step #1.5 - Add unit tests

### ISSUE THREAD

**Issue by reykjalin** [Automattic/woocommerce-payments#839]
@RadoslavGeorgiev - please open a separate PR for unit tests - I hate to do it, but let's ship this without the unit tests so that the rest of the subscriptions work is as unblocked as possible - we'll commit to add the tests later this sprint or next one - please add the unit test issue to sprint 18 backlog post - thx!

_Originally posted by @allendav in https://github.com/Automattic/woocommerce-payments/pull/826#issuecomment-679416440_

cc @allendav
